mavros_extras/CMakeLists.txt

```diff
@@ -135,10 +136,10 @@ target_link_libraries(gcs_image_bridge
   ${catkin_LIBRARIES}
 )
 
-add_executable(copter_visualization
-  src/copter_visualization.cpp
+add_executable(visualization
```

Comment:
But not just copter visualizer. Now it has also landing targets. So it's more general.